app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt

```diff
@@ -1432,15 +1459,82 @@ class ChatController(args: Bundle) : BaseController(args), MessagesListAdapter
                         }
                         true
                     }
+                    R.id.action_delete_message -> {
+                        ncApi?.deleteChatMessage(
+                                credentials,
+                                ApiUtils.getUrlForMessageDeletion(conversationUser?.baseUrl, roomToken, message?.id)
+                        )?.subscribeOn(Schedulers.io())
+                                ?.observeOn(AndroidSchedulers.mainThread())
+                                ?.subscribe(object : Observer<ChatOverallSingleMessage> {
+                                    override fun onSubscribe(d: Disposable) {
+                                    }
+
+                                    override fun onNext(t: ChatOverallSingleMessage) {
+                                        if (t.ocs.meta.statusCode == 202) {
```

Comment:
202 is hard to remember. Can you replace this by a const so it can have a good name?
Maybe there is already such a http status in http library.

Reply:
yep
java.net.HttpURLConnection.HTTP_ACCEPTED